apps/nextra/pages/en/build/sdks/ts-sdk/quickstart.mdx

```diff
@@ -2,60 +2,11 @@
 title: "TypeScript SDK Quickstart"
 ---
 
-import { Callout, Steps, Tabs } from 'nextra/components'
+import { Callout, Steps } from 'nextra/components'
 
 # TypeScript SDK Quickstart
 
-<Callout type="info">
-The complete example code can be found at [Full Quickstart Code](#Full_Quickstart_Code) at the bottom of the page.
-</Callout>
-
```

Comment:
The changes on this file seem like mistakes... Can you rebase again and remove these changes?